components/konflux-workspaces/base/operator/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
```

Comment:
the kustomize build is failing because you change the namespace in `/components/workspaces/base/operator/config/manager/manager.yaml`. Either revert to system so kustomize can apply the current patch matching the deployment in system NS or change the namespace here to `konflux-workspaces`